### DISCUSSION TITLE
vite build with react markdown vfile

### DISCUSSION THREAD

**Discussion by SangVoM** [remarkjs/remark#1355] [Q&A] [answered]
<img width="372" alt="Screenshot 2024-08-14 at 14 09 37" src="https://github.com/user-attachments/assets/35463cb6-2ae5-4864-8904-514e0e053353">

I am getting this error with the file when building it with vite please tell me how to fix it
why do i think it is react-markdown because in my project only react-markdown uses vfile

<img width="709" alt="Screenshot 2024-08-14 at 14 14 09" src="https://github.com/user-attachments/assets/1a620761-0e70-48d1-8b7d-3e4f2efb7a28">

**Comment by wooorm** [comment, marked as answer]
Same as https://github.com/orgs/remarkjs/discussions/1356, update your things

**Reply by SangVoM** [reply]
ok thanks
